projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-js/pdf-js-wrapper.ts

```diff
@@ -116,10 +116,14 @@ export class PdfJsWrapper {
     return this.pdfViewer.pagesRotation = (this.pdfViewer.pagesRotation + rotation) % 360;
   }
 
-  public getRotate(): number {
+  public getPagesRotation(): number {
     return this.pdfViewer.pagesRotation;
   }
 
+  public getNormalisedPagesRotation(): number {
```

Comment:
why have both of these methods, surely we just need one

Reply:
I'll remove the getPagesRotation